liwords-ui/src/profile/profile.tsx

```diff
@@ -238,105 +244,111 @@ export const PlayerProfile = React.memo(() => {
     offset: number;
     array: Array<GameMetadata>;
   }>({ numGames: gamesPageSize, offset: 0, array: [] });
-  const { loginState } = useLoginStateStoreContext();
-  const { username: viewer } = loginState;
   const [recentGamesOffset, setRecentGamesOffset] = useState(0);
   const [missingBirthdate, setMissingBirthdate] = useState(true); // always true except for self
 
   useEffect(() => {
-    axios
-      .post<ProfileResponse>(
-        toAPIUrl('user_service.ProfileService', 'GetProfile'),
-        {
-          username,
-        }
-      )
-      .then((resp) => {
-        setUserFetched(true);
-        setMissingBirthdate(!resp.data.birth_date);
-        setRatings(JSON.parse(resp.data.ratings_json).Data);
-        setStats(JSON.parse(resp.data.stats_json).Data);
-        setUserID(resp.data.user_id);
-        setCountryCode(resp.data.country_code);
-        setFullName(resp.data.full_name);
-        setAvatarUrl(resp.data.avatar_url);
-        setAvatarsEditable(resp.data.avatars_editable);
-        setBio(resp.data.about);
-        setBioLoaded(true);
-      })
-      .catch((e: AxiosError) => {
-        setUserFetched(true);
-        errorCatcher(e);
-      });
+    if (username) {
```

Comment:
let's do `if (!username) { return }` in this and the other one to make the diff a little nicer?